src/eval.rs

```diff
@@ -33,10 +36,14 @@ pub fn create_ecx<'mir, 'tcx: 'mir>(
     let mut ecx = InterpCx::new(
         tcx.at(syntax::source_map::DUMMY_SP),
         ty::ParamEnv::reveal_all(),
-        Evaluator::new(),
+        Evaluator::new(config.communicate),
         MemoryExtra::new(StdRng::seed_from_u64(config.seed.unwrap_or(0)), config.validate),
     );
 
+    // Complete initialization.
```

Comment:
*sigh* why is there now also an empty line here...?
It's okay, I will do it myself, that's easier.^^

Reply:
Sorry I thought you wanted me to add both

Reply:
Seems like we need to work on our communication. :/
I never used the plural. Not sure what else to do, other than add a list of all the things I do *not* mean. ;)

Reply:
Hahaha ok. At first I thought you wanted the space above the `// Complete initialization.` comment:
> Please leave an empty line above.
> 
> Code is a lot like text: paragraph help to structure it.
Then you said:
> You resolved this, I can't see the empty line (above "setup first stack frame")?
So I added both.

Reply:
But [above](https://github.com/rust-lang/miri/pull/894#discussion_r313967349) was a comment attached to the "first stack frame" comment. That is the line it referred to.

Reply:
Yeah anyway it's my bad. I just wanted to let you know I had somewhat a reason to do it